source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,362 @@
+.. _realm-sync-migrate-modes:
+
+=========================
+Migrate Device Sync Modes
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+Atlas App Services Apps can have one of two Device Sync modes: Partition-Based Sync
+and Flexible Sync. Partition-Based Sync is an older mode and you should consider
```

Comment:
We use "is an older mode" in other places throughout the docs. I think we were hedging our bets a bit because we weren't sure what product's EOL plan for Partition-Based Sync is. Has it been communicated anywhere other than docs that PBS may be deprecated?

Reply:
What about "Partition-Based Sync may be deprecated in the future and you should..."

Reply:
Should you also include ref links for more information about the two modes?

Reply:
I don't think docs are a good place to mention that PBS may be deprecated in the future. Unless product has an EOL plan I haven't heard of (entirely possible), it's pure speculation at this point.

Good point about adding ref links. I'll add those.

Reply:
I can follow up with Ian when he's back from vacation.

Reply:
We do have a ticket to disallow enabling PBS on new sync apps which effectively makes it "deprecated", but I agree this is maybe not the right place for users to be hearing this for the first time